docs/monitoring/autodiscovery.md

```diff
@@ -90,6 +90,30 @@ In container environments you add the `MONITORING_TOKEN` environment variable, o
 
 In all environments this makes it possible to use different monitoring tokens, and thus different Sematext Apps for the same service type running on the same machine or in the same Kubernetes/Swarm cluster.
 
+### Switching from manual to automatic monitoring
+
+The switch from manual to automatic monitoring is done by first removing manually set up agent and then enabling automatic monitoring in the user interface.
+
+In bare-metal/virtual machine setups there are two types of agents - in-process (with javaagent) and standalone.
```

Comment:
Should we write things in a way that will be correct even when we start using V for monitoring?

Reply:
I'd rather we update these docs once we actually start using the Vector instead of trying to guess how it will be set up.

Reply:
Though, maybe you have some generic idea in mind that I just don't see? How did you think we would write the docs to achieve what you suggest and still make that info useful?

Reply:
I don't.  I just saw -javaagent and realized this will not be 100% accurate in not too distant future.  But we can leave if if you/we don't have clear ideas now because we we will have to sweep through the docs when the time comes to address this anyway.

Reply:
Yeah, let's then postpone.